src/librustc_resolve/lib.rs

```diff
@@ -2677,8 +2665,7 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
                                    path: &Path,
                                    namespace: Namespace,
                                    check_ribs: bool)
-                                   -> AssocItemResolveResult
-    {
+                                   -> AssocItemResolveResult {
```

Comment:
hmm I guess I really am going to have to go fix those emacs `rust-mode` indentation bugs that have led me to put in these line breaks before the `{`